message/message-http/src/main/java/com/mastercard/test/flow/msg/http/HttpReq.java

```diff
@@ -75,8 +75,7 @@ public HttpReq( byte[] content, Function<byte[], Message> bodyParse ) {
 		while( !lines.isEmpty() && !(headerLine = lines.removeFirst()).isEmpty() ) {
 			Matcher h = HEADER_LINE_PATTERN.matcher( headerLine );
 			if( h.matches() ) {
-				set( HEADER_PRESUFIX + h.group( "name" ).trim() + HEADER_PRESUFIX,
-						h.group( "value" ).trim() );
+				set( header( h.group( "name" ).trim() ), h.group( "value" ).trim() );
```

Comment:
Removing these ad-hoc reimplementations of the `header` method in favour of just using the real thing